canonical-web-and-design/cookie-policy
# Added CSS to hide and show notification

Added CSS to hide and show if a browser does not support dialogs.

## QA
- Pull this code
- Run `gulp build`
- Open `index.html`
- Click the X and check it works
- Now run a local server
- On an apple phone click the X and see it disappear or wait 3 seconds and it will.

### Details
Fixes https://github.com/canonical-websites/www.canonical.com/issues/195
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/84
